projects/common/recaptcha/src/recaptcha.component.html

```diff
@@ -0,0 +1,2 @@
+<re-captcha (resolved)="resolved($event)" siteKey="6LfHcJcbAAAAAJA_kkeR4AXt92hSUpCxb-mKeWkT">
```

Comment:
We definitely don't want to store the `siteKey` in the source code. It should be passed to the component from an environment variable from "spa-env-server".

Reply:
The siteKey is a public key.  The public key is in the UI component, and the private key is an env variable to the server side recaptcha-service.
I can't fully demo the final comunication with the recaptcha service until it is in the openshift environment, which requires it to be integrated into an appication(BCP).  To do this it must be in the main branch of the common library.

Reply:
1. So does that mean it's ok to share the same `siteKey` for all projects? I thought Google limits the number of captcha requests/verifications each key can have before they start charging money for it's use. If that's the case, then we probably want the ability to have different public keys too.

2. Ok, but does MSP for example still run with the library linked to it locally?

Reply:
1) we are likely to need to pay for a key set, but we will need to discuss that with Jam and work out how we want it to work for licensing.
2) MSP still works with no changes when linked to the library locally.
The recaptcha component displays and workes to the point of attempted communictation with the service on my branch of BCP when linked locally, but until it is on openshift I can't test the link between component and service.

Reply:
Ok thanks, sounds good.